openshift_cli_installer/libs/clusters/ipi_cluster.py

```diff
@@ -111,6 +113,12 @@ def _create_install_config_file(self):
         with open(os.path.join(self.cluster_info["cluster-dir"], "install-config.yaml"), "w") as fd:
             fd.write(yaml.dump(cluster_install_config))
 
+    def _set_docker_config_file(self):
+        docker_config_dir = tempfile.mkdtemp()
```

Comment:
you need to delete the dir at the end of the flow

Reply:
updated the code to remove this dir